sensor/common/clusterentities/debug.go

```diff
@@ -8,6 +8,7 @@ import (
 
 	"github.com/pkg/errors"
 	"github.com/stackrox/rox/pkg/concurrency"
+	"golang.org/x/exp/maps"
```

Comment:
I think, it should be
```suggestion
	"maps"
```

Reply:
Done

Reply:
Thanks!